### ISSUE TITLE
Problem with loader/add-dependencies! on Windows in boot-1.0.3

### ISSUE THREAD

**Issue by cpmcdaniel** [tailrecursion/boot#23]
When following the getting started guide for setting up boot-1.0.3 on Windows, I get the following error when passing the :strap argument in an empty directory. The .boot/boot-classloader.jar file is created, but dependency resolution soon fails:

```
java.io.FileNotFoundException: Could not locate ancient_clj/core__init.class or ancient_clj/core.clj on classpath:    
 at clojure.lang.RT.load (RT.java:443)                                                                                
    clojure.lang.RT.load (RT.java:411)                                                                                
    clojure.core$load$fn__5018.invoke (core.clj:5530)                                                                 
    clojure.core$load.doInvoke (core.clj:5529)                                                                        
    clojure.lang.RestFn.invoke (RestFn.java:408)                                                                      
    clojure.core$load_one.invoke (core.clj:5336)                                                                      
    clojure.core$load_lib$fn__4967.invoke (core.clj:5375)                                                             
    clojure.core$load_lib.doInvoke (core.clj:5374)                                                                    
    clojure.lang.RestFn.applyTo (RestFn.java:142)                                                                     
    clojure.core$apply.invoke (core.clj:619)                                                                          
    clojure.core$load_libs.doInvoke (core.clj:5413)                                                                   
    clojure.lang.RestFn.applyTo (RestFn.java:137)                                                                     
    clojure.core$apply.invoke (core.clj:619)                                                                          
    clojure.core$require.doInvoke (core.clj:5496)                                                                     
    clojure.lang.RestFn.invoke (RestFn.java:408)                                                                      
    tailrecursion.boot.strap$latest_version.invoke (strap.clj:14)                                                     
    tailrecursion.boot.strap$strap.invoke (strap.clj:22)                                                              
    tailrecursion.boot.loader$_main.doInvoke (loader.clj:175)                                                         
    clojure.lang.RestFn.applyTo (RestFn.java:137)                                                                     
    tailrecursion.boot.loader.main (:-1)                                                                             
```

Hacking around in the boot source code, I found that the URL paths for dependencies should be prefixed with **file:///** (triple-slashed), not **file://** (double-slashed). The URI class seems stricter. If you try to load the latter into a new URI instance, it will fail to parse it. 

The java.net.\* API seems to suggest constructing a URI, then calling .toURLI() on it, then calling .getFile().

**Comment by cpmcdaniel** [comment]
I'm not sure, but it is possible the fix may positively affect bugs #10 and #13

**Comment by cpmcdaniel** [comment]
Unfortunately, URI doesn't like backslashes in the string. I suppose we should stick with the URL class. Simply adding the third slash to the protocol prefix does the trick.
